frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/switch-to-form-validation-utils.ts

```diff
@@ -0,0 +1,110 @@
+import i18n from 'i18next';
+import * as yup from 'yup';
+import { nameValidationSchema } from '@console/shared';
+import { PipelineTask } from '../../../types';
+import { PipelineBuilderFormValues } from './types';
+import { runAfterMatches } from './validation-utils';
+
+const resourceDefinitionYAML = () => {
+  return yup.array().of(
+    yup.object({
+      name: yup.string(),
```

Comment:
Probably, you need to this one also a required field to remove invalid resources.

Reply:
yes. I could also reproduce the issue for task parameters. updated that as well.
Thanks for catching this.